protocol/strict-derivation.md

```diff
@@ -0,0 +1,269 @@
+# Purpose
+
+Holocene aims to introduce a set of changes to derivation that improve worst-case scenarios for
+Fault Proofs and Interop. The goal of this design doc is to align on the changes to derivation that
+will be part of the upcoming Holocene hardfork, and in particular to solve open questions to
+finalize the specs.
+
+The Holocene specs for these derivation changes are currently being written in [specs/#357](https://github.com/ethereum-optimism/specs/pull/357).
+
+# Summary
+
+The Holocene hardfork introduces several changes to batch derivation:
+
+- _Partial Span Batch Validity_ determines the validity of singular batches within a span batch
+individually, only invalidating the remaining span batch upon the first invalid batch.
+- _Steady Batch Derivation_ derives invalid singluar batches immediately as deposit-only
```

Comment:
```suggestion
- _Steady Batch Derivation_ derives invalid singular batches immediately as deposit-only
```

Reply:
fixed in https://github.com/ethereum-optimism/design-docs/pull/73 right?